bot/telegram_bot.py

```diff
@@ -36,36 +36,47 @@ def __init__(self, config: dict, openai: OpenAIHelper):
         self.openai = openai
         bot_language = self.config['bot_language']
         self.commands = [
-            BotCommand(command='help', description=localized_text('help_description', bot_language)),
-            BotCommand(command='reset', description=localized_text('reset_description', bot_language)),
-            BotCommand(command='image', description=localized_text('image_description', bot_language)),
-            BotCommand(command='stats', description=localized_text('stats_description', bot_language)),
-            BotCommand(command='resend', description=localized_text('resend_description', bot_language))
+            BotCommand(command='help', description=localized_text(
+                'help_description', bot_language)),
```

Comment:
There are lots of changes like this adding unneeded newlines. Why is that? These are not required

Reply:
Made the PR long time ago.